roles/easy-build-packages/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+is_head: no
+is_compute: no
+
+eb_packages_path: "{{ eb_prefix_path}}/ebfiles_repo"
+eb_packages:
```

Comment:
Should use an empty list here `[]` and move the defaults to `config/group_vars/`